src/Process/CloudConfig.php

```diff
@@ -0,0 +1,173 @@
+<?php
```

Comment:
This class shouldn't be in Process namespace. In the `ece-tools` we are using Processes as some stage during build/deploy/post-deploy hooks. Let's move this class to something like `Magento\MagentoCloud\Command\CloudConfig\Renderer`.

P.S. some commands are using processes but just because they are using in build/deploy hooks too.